### DISCUSSION TITLE
Toolbar contextual view

### DISCUSSION THREAD

**Discussion by drewvolz** [StoDevX/AAO-React-Native#6039] [Ideas]
From @hawkrives: https://github.com/StoDevX/AAO-React-Native/issues/6035#issuecomment-1117660535   

> I'm thinking that it'd be nice to put a small toolbar at the bottom, like Mail.app has, on any view without a tab bar. I think it would show the last-updates time and maybe an (i) icon to tell you where we get the data from (IE it would replace the current ListFooterComponent).

**Comment by hawkrives** [comment]
This would solve the problem of SafeAreaView, at the expense of ~65% of a row's height. We wouldn't need one on any view with a tab bar, either.

**Reply by drewvolz** [reply]
I think that's true for portrait mode. In landscape we'd still need safearea to not run into the notch on both sides.

**Comment by drewvolz** [comment]
@hawkrives did the recent toolbar work do this?

**Reply by hawkrives** [reply]
Not yet. I'm now envisioning the "last updated" info being at the bottom of the per-screen menu, near Report a Problem? Or maybe we could show it when you PTR - "last checked\n Last changed" or something
